### PR TITLE
One more

### PR DESCRIPTION
**Pull request by Tayyab94**

### CODE REVIEW

**Comment by Tayyab94** [comment]
Olease check this one

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,6 @@
 !.css/index.js
 .css/index.css
 
-log.txt
\ No newline at end of file
+log.txt
+helper/employee.css
+!helper/user.css
```
Comment (Tayyab9) [inline .gitignore:6]: Here are facing issue

**Review by Tayyab9 — APPROVED** [review]

**Comment by Tayyab94** [comment]
Check please

**Review by Tayyab9 — APPROVED** [review]

**Review by Tayyab9 — APPROVED** [review]

**Comment by Tayyab94** [comment]
Please Accept it. i have releaver Entities

**Review by Tayyab9 — APPROVED** [review]

**Comment by Tayyab9** [comment]
WelDone dev.

**Comment by Tayyab94** [comment]
Now now

**Review by Tayyab9 — APPROVED** [review]

**Review by Tayyab9 — APPROVED** [review]

**Pull request merged**